notebooks/ch11/gradient_stochastic.ipynb

```diff
@@ -7,7 +7,8 @@
     "ExecuteTime": {
```

Comment:
To calculate the minimizing value of θ -> $\hat{\theta_0}$

Reply:
Overall, in this file, it looks theta was changed to \hat{theta} instead of \hat{theta_0}. The reason is that we're just looking at a population statistic, so we're estimating the theta_0.